x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/installed_integrations/components/confirm_bulk_uninstall_modal.tsx

```diff
@@ -53,14 +71,12 @@ export const ConfirmBulkUninstallModal: React.FunctionComponent<{
         title={i18n.translate('xpack.fleet.installedIntegrations.bulkUninstallModal.calloutTitle', {
           defaultMessage: 'This action cannot be undone.',
         })}
-        content={i18n.translate(
-          'xpack.fleet.installedIntegrations.bulkUninstallModal.calloutContent',
-          {
-            defaultMessage:
-              'All Kibana and Elasticsearch assets created by these integrations will be also removed. Review and edit your selection if needed.',
-          }
-        )}
-      />
+      >
+        <FormattedMessage
+          id="xpack.fleet.installedIntegrations.bulkUninstallModal.calloutContent"
+          defaultMessage="All Kibana and Elasticsearch assets created by these integrations will be also removed. Review and edit your selection if needed."
```

Comment:
For single selection, should we say `All Kibana and Elasticsearch assets created by this integration will be also removed.`